crawler/file_processing.py

```diff
@@ -400,19 +459,51 @@ def process_samples(self, add_to_dart: bool) -> None:
                     filter(lambda x: x[FIELD_MONGODB_ID] in mongo_ids_of_inserted, docs_to_insert)
                 )
 
+                # Get root sample ids of samples that were sucessfully inserted into the samples collection
+                root_sample_ids = self.get_root_sample_ids_from_samples_collection(mongo_ids_of_inserted)
+
+                important_unprocessed_priority_samples = self.get_important_unprocessed_priority_samples(
+                    root_sample_ids
+                )
```

Comment:
get_important_unprocessed_priority_samples should receive mongo_ids_of_inserted, not work with root sample id